recipes-core/everest-chargebyte/everest-chargebyte_git.bb

```diff
@@ -20,4 +20,9 @@ INSANE_SKIP:${PN} = "already-stripped useless-rpaths arch file-rdeps"
 
 FILES:${PN} += "${datadir}/everest/*"
 
-EXTRA_OECMAKE += "-DDISABLE_EDM=ON"
+EXTRA_OECMAKE += " \
+    -DDISABLE_EDM=ON \
+    -Deverest-chargebyte_INSTALL_EV_CLI_IN_PYTHON_VENV=OFF \
+    -Deverest-chargebyte_USE_PYTHON_VENV=OFF \
```

Comment:
I would restest it with only -Deverest-chargebyte_USE_PYTHON_VENV=OFF and removing the other 2 flags related to Python Venv

Reply:
It works with the removed variables. I will squash the commit which changes this and then merge.